calour/io.py

```diff
@@ -466,6 +382,90 @@ def read_amplicon(data_file, sample_metadata_file=None,
     return exp
 
 
+@ds.with_indent(4)
+def read_open_ms(data_file, sample_metadata_file=None, gnps_file=None, feature_metadata_file=None,
```

Comment:
this whole function is relocated? it's hard to see if there is any changes...

Reply:
just relocated since i think otherwise docscript has problems since the function for which we need the kwargs parameters list is after the function that needs to print them.
not changes otherwise